src/server/helpers/error.js

```diff
@@ -19,14 +19,21 @@
 'use strict';
 
 const status = require('http-status');
+const React = require('react');
+const ReactDOMServer = require('react-dom/server');
+const propHelpers = require('./props');
+const Layout = require('../../client/containers/Layout');
+const ErrorPage = require('../../client/components/pages/error');
 
 class SiteError extends Error {
-	constructor(message) {
+	constructor(message, req) {
```

Comment:
Hmm, I feel like we should only provide `req` as a parameter to error classes that are route-specific. So, for example, add a constructor to those classes which takes `message` and `req` parameters, calls `super(message)` and then creates the detailed message.

This'll allow us to keep using SiteError for things like errors during the app startup (outside of requests).